nbs/11_commands.ipynb

```diff
@@ -63,11 +63,19 @@
     "        filename = os.path.join(curpath, relpath)\n",
     "\n",
     "        subprocess.run(['streamlit', 'run', filename] + sys.argv[1:])\n",
-    "    return run_streamlit_fn_fn\n",
-    "\n",
-    "\n",
+    "    return run_streamlit_fn_fn"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#| exporti\n",
+    "#| eval: false\n",
     "# Run explorer app\n",
-    "run_explorer = streamlit_fn_factory('./explorer.py',os.path.dirname(__file__))"
+    "run_explorer = streamlit_fn_factory('./explorer.py',os.getcwd())"
```

Comment:
This change currently breaks stk_explorer as a CLI command.

Why is it necessary anyway?